prereise/gather/griddata/hifld/const.py

```diff
@@ -1,3 +1,5 @@
+s_base = 100  # MVA
```

Comment:
Why not using `baseMVA` as we do in **PowerSimData**?

Reply:
Good idea.

Reply:
Done. We also currently use `s_base` elsewhere in PreREISE though, and it follows a more pythonic naming convention, whereas I think the `baseMVA` in PowerSimData originates with Matpower. Not sure which approach is best.

Reply:
Yes it originates from Matpower. If we already use `s_base` elsewhere in PreREISE then my consistency argument is no longer valid and I let you make an executive decision